packages/web/hooks/data/data-filter.ts

```diff
@@ -13,6 +13,7 @@ export class DataFilter<TData> implements DataProcessor<TData[]> {
       keys: keys,
       findAllMatches: true,
       useExtendedSearch: true,
+      threshold: 0.2,
```

Comment:
maybe leave a comment here

Reply:
Thanks for the feedback! I added the comment on my last commit